src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -279,6 +279,9 @@ java.launcher.cls.error6=\
 java.launcher.cls.error7=\
     Error: non-static inner class {0} constructor can not be invoked \n\
     make inner class static or move inner class out to separate source file
+java.launcher.cls.error8=\
+    Error: abstract class {0} can not instantiated\n\
```

Comment:
Nit - should this use a comma instead of the newline?

Reply:
The other errors here provide some advice on the second line, so I kept that pattern for this error as well. So, the newline is intentional.

Reply:
Sounds fine to me then.